armi/reactor/tests/test_components.py

```diff
@@ -85,6 +91,12 @@ def test_factory(self):
         self.assertIsInstance(fuelComp.material, materials.UZr)
 
     def test_componentInitializationAndDuplication(self):
+        """Initialize and duplicate a component, veifying the parameters.
```

Comment:
I don't really understand what this test is trying to prove.

It just creates a bunch of components and then verifies that they can be `deepcopy`-ed? Maybe you can add some clarification to the docstring about what this is supposed to prove?

Reply:
You're right....I think the only thing it proves is that a component has parameters...

Reply:
Correct. The requirement says that a component must have parameters, so this test covers that.  (So do many others, but I picked one.)

The test above proves that components have attributes. That's from the same req.

Do we need to add to the docstring here?

Reply:
I think the docstring repeats what you just said, so it's probably fine?